changelog.txt

```diff
@@ -2,6 +2,12 @@
 
 Version 1.9.11, 1 February 2016
 
+The Checker Framework distribution now contains annotated versions
```

Comment:
Hi Mike. I just pushed a commit with some fixes. If you think they look good, I can create a clean branch with my changes in their current form and merge it tonight.
There isn't time tonight for me to make (and for us to agree on) the potential improvements to the README Werner asked about and the checker-includes/excludes related fix.
One thing I am unsure about is this line in the manual I uncommented:
\href{https://commons.apache.org/proper/commons-bcel/}{Apache Commons BCEL} (versions 5.2 and 6.0 pre-release)
Are these two still the versions that are checked in? Since you were involved with Dan and/or Mark in updating these, I thought you might know.